src/uploadcare-ckeditor4.js

```diff
@@ -1,100 +1,118 @@
-// Uploadcare CKeditor plugin
-// Version: 2.1.2
-
 CKEDITOR.plugins.add('uploadcare', {
   hidpi: true,
   icons: 'uploadcare',
-  init : function(editor) {
+  init: function(editor) {
     var config = editor.config.uploadcare || {};
 
     // Check if Uploadcare is already loaded and load it if not.
     if (typeof uploadcare === 'undefined') {
-        var version = config.widgetVersion || '2.x';
-        var widget_url = 'https://ucarecdn.com/libs/widget/' + version +
-                 '/uploadcare.full.min.js'
-        CKEDITOR.scriptLoader.load(widget_url);
+      var version = config.widgetVersion || '$_WIDGET_VERSION';
+      var widgetUrl = 'https://ucarecdn.com/libs/widget/' + version + '/uploadcare.full.min.js';
+
+      CKEDITOR.scriptLoader.load(widgetUrl);
     }
 
     // Apply default properties.
-    if ( ! ('crop' in config)) {
+    if (!('crop' in config)) {
       config.crop = '';
     }
 
+    applyIntegrationSetting();
+
+    function applyIntegrationSetting() {
+      var editorVersion = CKEDITOR.version;
+      var pluginVerion = '$_VERSION';
```

Comment:
@dmitry-mukhin it comes from `package.json` via `rollup-plugin-jscc` (something like c/c++ preprocessor)
It will be replaced during the build step, output will be deployed to `releases` branch and to the releases page.

Reply:
we need to have an instruction how to release things then :)